docs/How_to_start_and_run_testcases.md

```diff
@@ -1,19 +1,191 @@
 # Description
 This is the directory to document how to start/run the tests from the scratch.
 
-Hardware Requirements :
+## Hardware Requirements
+
+  * 7 Dentos Devices.
+  * 1 Ixia Chassis with 16 10G port running IxOS/IxNetwork EA versions [We are using 9.20 EA].
+  * 1 IxNetwork EA API server.
+  * 1 linux with Ubuntu 22.04 (centos8 will also work or other distros but the instructions bellow are for ubuntu 22.04) 
+TODO: create a lab BOM
+
+## Prepare Testbed Server
+
+- Install Ubuntu[^1] 22.04 x64 on the server. ([ubuntu-22.04.1-live-server-amd64.iso](https://releases.ubuntu.com/22.04/))
+  - select all default options (unless otherwise noted bellow)
+  - on disk setup: disable LVM (optional)
+  - on profile setup: put name, servername, username, password all as `dent` for example purposes
+  - on ssh setup: enable `install OpenSSH server`
+- Install Ubuntu prerequisites
+    ```
+    sudo apt -y update
+    sudo apt -y upgrade
+    sudo apt -y autoremove
+    sudo apt -y install \
+      python3 \
+      python3-pip \
+      net-tools \
+      curl \
+      git \
+      make
+    sudo apt -y install ubuntu-desktop (TODO: remove this depedency)
+    ```
+- install Docker (all credits to [Docker manual](https://docs.docker.com/engine/install/ubuntu/) )
+    ```
+    sudo apt-get -y remove docker docker-engine docker.io containerd runc
+    sudo apt-get update
+    sudo apt-get -y install \
+      apt-transport-https \
+      ca-certificates \
+      curl \
+      gnupg-agent \
+      gnupg \
+      lsb-release \
+      software-properties-common
+    sudo mkdir -p /etc/apt/keyrings
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+    echo \
+      "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+      $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    sudo apt-get update
+    sudo apt-get -y install docker-ce docker-ce-cli containerd.io docker-compose-plugin
+    sudo docker run hello-world
+    ```
+    - add your user to docker group
+        ```
+        sudo usermod -aG docker $USER
+        ```
+ - install KVM

```

Comment:
Do I understand correctly that KVM is required by IxNetwork Web? Maybe it worth to mention this here.